test/jest/Bladeburner/BladeburnerConsole.test.ts

```diff
@@ -0,0 +1,92 @@
+import type { Bladeburner } from "../../../src/Bladeburner/Bladeburner";
+import { Player, setPlayer } from "@player";
+import { PlayerObject } from "../../../src/PersonObjects/Player/PlayerObject";
+import { BladeburnerContractName } from "@enums";
+import { Contract } from "../../../src/Bladeburner/Actions";
+
+describe("Bladeburner Console", () => {
+  let inst: Bladeburner;
+  const FEEDBACK_CMDS = [["skill"], ["start"], ["help"], ["log"], ["automate"]];
+  const SILENT_CMDS = [["cls"], ["clear"], ["stop"]];
```

Comment:
Optional-ish: I think it would be more clear to have these lists directly inside the `each`, so it is more clear what exactly is being tested. This is a general principle in tests; it is often worth it to be more verbose (sometimes to the point of duplication) in order to get extreme clarity.

At the very least, `SILENT_CMDS` is only used once, so really should be inlined.

Reply:
I kept the alias for clr in the actual functionality check, and dropped the help check.